src/api/routes/roles.routes.js

```diff
@@ -92,7 +125,7 @@ roleRouter.post("/", adminCheck, validateCreateRole, createRole);
  *       500:
  *         description: Internal server error
  */
-roleRouter.get("/:role", getRoleById);
+roleRouter.get("/id/:id", roleExistsById, getRoleById);
```

Comment:
why /id/:id

Reply:
I added two middlewares. one of getting role by Id and another of getting role by title. Technically that's how to get the two separated 

`roles/id/[role-id]`

`roles/title/[role-title]`